### DISCUSSION TITLE
RFC - remote development module architecture

### DISCUSSION THREAD

**Discussion by teronslair** [lapce/lapce#2609] [General]
Hello dev team,

Nice work so far!
I was a bit curious about the remote development module.
I think this is quite a valuable "selling point", and as a daily VScode user of that feature I couldn't be happier to see alternatives.

Can you briefly explain your solution architecture for this feature?
What processes are running on each machine, what data exchange takes place, maybe an overview of the initial supported operations in this mode?

Cheers,
Razvan

**Comment by panekj** [comment]
https://docs.lapce.dev/development/architecture

**Reply by teronslair** [reply]
Crystal clear! Case closed (sorry I missed the link to the official page).

**Comment by teronslair** [comment]
A classic case of RTFM where the Manual location was not immediately apparent.
